src/shared/components/msktable/styles.scss

```diff
@@ -4,4 +4,16 @@ th.sort-des:after {
 
 th.sort-asc:after {
   content: ' \25B2';
-}
\ No newline at end of file
+}
+
+.topPagination {
+  .textBetweenButtons {
+    opacity: 1.00 !important;
```

Comment:
Since the text between the buttons is actually a "disabled button", it doesn't have full opacity. This restores the text to its full opacity